contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -145,4 +157,230 @@ contract OETHVaultCore is VaultCore {
 
         _postRedeem(_amount);
     }
+
+    /**
+     * @notice Request an asynchronous withdrawal of WETH in exchange for OETH.
+     * The OETH is burned on request and the WETH is transferred to the withdrawer on claim.
+     * This request can be claimed once the withdrawal queue's `claimable` amount
+     * is greater than or equal this request's `queued` amount.
+     * There is no minimum time or block number before a request can be claimed. It just needs
+     * enough WETH liquidity in the Vault to satisfy all the outstanding requests to that point in the queue.
+     * OETH is converted to WETH at 1:1.
+     * @param _amount Amount of OETH to burn.
+     * @param requestId Unique ID for the withdrawal request
+     * @param queued Cumulative total of all WETH queued including already claimed requests.
+     */
+    function requestWithdrawal(uint256 _amount)
+        external
+        whenNotCapitalPaused
+        nonReentrant
+        returns (uint256 requestId, uint256 queued)
+    {
+        // Burn the user's OETH
+        // This also checks the requester has enough OETH to burn
+        oUSD.burn(msg.sender, _amount);
+
+        requestId = withdrawalQueueMetadata.nextWithdrawalIndex;
+        queued = withdrawalQueueMetadata.queued + _amount;
+
+        // store the next withdrawal request
+        withdrawalQueueMetadata.nextWithdrawalIndex = uint128(requestId + 1);
+        withdrawalQueueMetadata.queued = uint128(queued);
+
+        emit WithdrawalRequested(msg.sender, requestId, _amount, queued);
+
+        withdrawalRequests[requestId] = WithdrawalRequest({
+            withdrawer: msg.sender,
+            claimed: false,
+            timestamp: uint40(block.timestamp),
+            amount: uint128(_amount),
+            queued: uint128(queued)
+        });
+    }
+
+    /**
+     * @notice Claim a previously requested withdrawal once it is claimable.
+     * This request can be claimed once the withdrawal queue's `claimable` amount
+     * is greater than or equal this request's `queued` amount.
+     * There is no minimum time or block number before a request can be claimed. It just needs
```

Comment:
The comment seems invalid since it doesn't account for `CLAIM_DELAY`

Reply:
Good spot. Fixed. Thanks